native/common/jp_proxy.cpp

```diff
@@ -14,8 +14,14 @@
    limitations under the License.
    
 *****************************************************************************/   
-#include <jpype.h>

```

Comment:
Many of the warnings were coming from jpype being above Python.h in files. These were all corrected.